hack/verify-rbac.sh

```
@@ -0,0 +1,74 @@
+#
+# Verbs NOT granted (and should NOT appear in production code):
+#   Update(), Delete(), Create()
+
+DISALLOWED_VERBS=(
+  "Nodes().Update("
+  "Nodes().Delete("
+  "Nodes().Create("
+)
+
+FOUND=0
+EXCLUDE_DIRS="-not -path '*/vendor/*' -not -path '*/.git/*'"
```

Shellcheck (0.11.0)

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)